app/front/scripts/directives/visualizations/piechart/template.html

```diff
@@ -8,6 +8,7 @@
     state="state"
     format-value="formatValue"
     messages="messages"
-    endpoint="{{ params.babbageApiUrl }}">
+    endpoint="{{ params.babbageApiUrl }}"
+    max-slices="5">
```

Comment:
@akariv I just merged https://github.com/openspending/babbage.ui/pull/62, now we need to release a new version. Do we have any document on how to do it, or shall I simply follow whatever was done before?